src/pages/Club/ClubListPage/index.tsx

```diff
@@ -1,47 +1,104 @@
-import { CLUB_CATEGORY } from 'static/club';
+import { cn } from '@bcsdlab/utils';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import useClubsCategories from 'pages/Club/hooks/useClubsCategories';
+import useClubList from 'pages/Club/hooks/useClubList';
+import { Selector } from 'components/ui/Selector';
 import styles from './ClubListPage.module.scss';
 
-function ClubCard() {
-  return (
-    <div>
-      <div>
-        <div>BCSD</div>
-        <div>학술분과</div>
-        <div>동아리소개최대33글자</div>
-      </div>
-      <div>image</div>
-    </div>
-  );
-}
+const sorts = [
+  { label: '생성순', value: '생성순' },
+  { label: '조회순', value: '조회순' },
+];
 
 function ClubListPage() {
-  const totalCount = 1;
+  const navigate = useNavigate();
+  const [sortValue, setSortValue] = useState(sorts[0].label);
+  const clubCategories = useClubsCategories();
+  const [selectedCategoryId, setSelectedCategoryId] = useState<number | undefined>(undefined);
```

Comment:
undefined는 명시해줄 필요가 없답니다.⭐️  없을 수 있다는 의미 전달을 해주고 싶다면 null을 써보는 것도 좋을 것 같아요!

```suggestion
  const [selectedCategoryId, setSelectedCategoryId] = useState<number>();
```

Reply:
헉...좋은 가르침..👍🏻